week3/project/ecommerce/src/components/Favorites.jsx

```diff
@@ -0,0 +1,37 @@
+import Navbar from "./Navbar";
+import { ProductContext } from "./ProductContext";
+import { useContext } from "react";
+import useFetch from "../hooks/useFetch";
+import ProductListCard from "./ProductListCard";
+
+const Favorites = () => {
+  const [favorites] = useContext(ProductContext);
+
+  const {
+    data: products,
+    loading,
+    error,
+  } = useFetch("https://fakestoreapi.com/products");
+
+  const filteredProducts = products.filter((product) =>
+    favorites.includes(product.id)
+  );
+
+  return (
+    <>
+      <Navbar title={"Favorites"} />
```

Comment:
Also here you can remove the curly braces 😄 

```suggestion
      <Navbar title="Favorites" />
```